### PR TITLE
Allow Comments on Pull Requests to be Optional

### PR DESCRIPTION
**Pull request by mtjandra**
* Add argument to allow comments on pull requests to be optional (`enabled` by default)
* Due to limitation in Travis, you also need to know what the master branch should be (`master` by default)

FYI: @AlexBedley

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -45,6 +47,11 @@ function comment( value, options, cb ) {
...
 	}
 
-	request.post( requestOptions, handleResponse );
-
+	if( includePullRequests ||
+			(process.env.TRAVIS_PULL_REQUEST === 'false' && process.env.TRAVIS_BRANCH === masterBranch ) ) {
+		request.post( requestOptions, handleResponse );
+	}
+	else {
+		console.log( 'Peanut gallery is configured not to comment on pull requests.' );
```
Comment (AlexBedley) [inline lib/index.js]: Should we invoke the [rule of silence](http://www.linfo.org/rule_of_silence.html) here?

**Comment by omsmith** [comment]
Why would you even invoke peanut-gallery if you don't want it to do things?

**Comment by mtjandra** [comment]
> Why would you even invoke peanut-gallery if you don't want it to do things?

We would still like peanut gallery to make comments when pull requests are merged to master.

**Comment by omsmith** [comment]
Right, but on your side, just don't invoke peanut-gallery in your Travis script unless you're on master.

**Comment by mtjandra** [comment]
> Right, but on your side, just don't invoke peanut-gallery in your Travis script unless you're on master.

Good point. Perhaps I was too excited to make a contribution towards peanut-gallery!

**Pull request closed without merging**